### PR TITLE
Bookings Data Model Requires Exam Received Date Field

### PR DESCRIPTION
**Pull request by akroon3r**
Bookings front end required a field that recorded the date-time stamp of when an exam was received in an office. To accomplish this, the exam_received_date field was added to the exam model and schema. Test data was modified to show that this field was added as well, and postman tests were updated to reflect this change as well. Postman tests were run to ensure that this change didn't fail any of the current tests.

### CODE REVIEW

**Comment by akroon3r** [comment]
Please note that I'm basing this branch off of SJRUMSBY MASTER, as basing it off sprint-2 includes commits that are not my own for some reason.

File: api/postman/bookings-postman.json [hunk 1]
```diff
@@ -0,0 +1,1983 @@
+{
```
Comment (sjrumsby) [inline api/postman/bookings-postman.json]: I'm guessing that this file shouldn't be part of this commit / PR?

File: api/postman/bookings-env-postman.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (sjrumsby) [inline api/postman/bookings-env-postman.json]: I'm also guessing that this file shouldn't be part of this commit / PR?

File: api/postman/bookings-postman.json (hunk 1 above)
Comment (akroon3r) [reply api/postman/bookings-postman.json]: No I updated the file to include the new field I added.

File: api/postman/bookings-env-postman.json (hunk 2 above)
Comment (akroon3r) [reply api/postman/bookings-env-postman.json]: No I updated the file to include the new field I added.

**Comment by akroon3r** [comment]
We should probably commit the integration testing PR first before this one?

**Comment by sjrumsby** [comment]
Yea, we'll need to. I just merged the other ticket, so now we need some resolution here.

**Comment by akroon3r** [comment]
Done and done.
